src/category_theory/limits/has_limits.lean

```diff
@@ -212,11 +223,12 @@ The isomorphism (in `Type`) between
 morphisms from a specified object `W` to the limit object,
 and cones with cone point `W`.
 -/
-def limit.hom_iso (F : J ⥤ C) [has_limit F] (W : C) : (W ⟶ limit F) ≅ (F.cones.obj (op W)) :=
+def limit.hom_iso (F : J ⥤ C) [has_limit F] (W : C) :
+  ulift.{u₁} (W ⟶ limit F : Type v) ≅ (F.cones.obj (op W)) :=
```

Comment:
Will we also still need a version without `ulift` for easy use when the universe variables are the same?

Reply:
Based on the use case downstream, I think this works fine when composing with `ulift_trivial`.